{"Timestamp":"2020-03-06T00:24:12.1839521+07:00","Level":"Error","MessageTemplate":"{\"Identity\":\"202003050524121837515176967\",\"TransactionId\":\"\",\"Server\":\"192.168.43.51\",\"Caller\":null,\"Client\":\"\",\"Location\":{\"ClassName\":\"CountryService\",\"Namespace\":\"Pelo.v2.Web.Services.Country\",\"Function\":\"GetAll\"},\"Message\":\"Exception of type 'Pelo.Common.Exceptions.PeloException' was thrown.\",\"Data\":{\"Parameters\":[],\"ReturnValue\":null},\"LogType\":\"Exception\",\"Elapsed\":670,\"ExecDate\":\"2020-03-06T00:24:12.1837778+07:00\",\"Site\":null}"}
